junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/NodeTestTask.java

```diff
@@ -165,13 +165,25 @@ private void cleanUp() {
 
 	private void reportCompletion() {
 		if (throwableCollector.isEmpty() && skipResult.isSkipped()) {
+			try {
+				node.nodeSkipped(context, testDescriptor, skipResult);
+			}
+			catch(Throwable t) {
+				//swallow
```

Comment:
If `nodeSkipped()` throws an exception, let's at least log it a `DEBUG` level.

Reply:
Swallowing `Throwable` is not allowed in JUnit 5. We need to rethrow it if it's _blacklisted_.